common/src/thrift/tachyon.thrift

```diff
@@ -26,7 +26,7 @@ struct WorkerInfo {
 struct BlockLocation {
   1: i64 workerId
   2: NetAddress workerAddress
-  3: i32 tier
+  3: string tierAlias
```

Comment:
We should define what's the difference between `tier` and `tierAlias` (if there is) because we have variables like `tierAlias` and then `usedBytesOnTiers`